tests/python/unittest/test_operator.py

```diff
@@ -6527,12 +6527,64 @@ def check_bilinear_resize_op(shape, height, width):
         x = mx.nd.random.uniform(shape=shape)
         y = mx.nd.contrib.BilinearResize2D(x, height=height, width=width)
         assert_almost_equal(y.asnumpy(), py_bilinear_resize(x.asnumpy(), height, width))
+    def check_bilinear_resize_modes_op(shape, height_factor=None, width_factor=None, shape_1=None, mode=None):
+        x = mx.nd.random.uniform(shape=shape)
```

Comment:
unit tests for operators should cover forward pass and backward pass. See here - https://mxnet.incubator.apache.org/faq/add_op_in_backend.html#unit-test

Reply:
Done :)